packages/shared/lib/core/network/interfaces/chain-metadata.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainType } from '../enums'
+
+export interface IChainMetadata {
```

Comment:
It might be good to rename this to `IBaseChainMetadata` so it's not confused with the `ChainMetadata` union type